addons/core/translations/resources/en-us.yaml

```diff
@@ -267,6 +267,8 @@ target:
       add:
         title: Add Host Sets
         description: Select host sets to assign to this target.
+  secret:
```

Comment:
Secret toggle seems pretty generic and could go with the actions translations.  Also, actions generally do not end with a period.

Reply:
Gotcha!